keras_hub/src/models/retinanet/retinanet_label_encoder_test.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2024 The KerasHub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import pytest
+import tensorflow as tf
+from keras import backend
+from keras import ops
+
+from keras_hub.src.models.retinanet.retinanet_label_encoder import (
+    RetinaNetLabelEncoder,
+)
+from keras_hub.src.tests.test_case import TestCase
+
+
+class RetinaNetLabelEncoderTest(TestCase):
+    def test_label_encoder_output_shapes(self):
+        images_shape = (8, 512, 512, 3)
```

Comment:
please keep image sizes for tests small maybe 64/128?

Reply:
Done